firebase-query.html

```diff
@@ -373,6 +391,17 @@
               });
             });
           }
+        },
+
+        __decoratedValue: function(value, key) {
```

Comment:
If this were `__snapshotToValue` or something, accepting a snapshot as its only argument, it might make the transformation being performed a little more explicit. WDYT?